google/cloud/bigtable/internal/mutate_rows_limiter.cc

```diff
@@ -85,6 +88,14 @@ std::shared_ptr<MutateRowsLimiter> MakeMutateRowsLimiter(CompletionQueue cq,
   std::function<void(duration)> sleeper = [](duration d) {
     std::this_thread::sleep_for(d);
   };
+  if (internal::Contains(options.get<TracingComponentsOption>(), "rpc")) {
+    sleeper = [sleeper = std::move(sleeper)](duration d) {
+      if (d != duration::zero()) {
+        GCP_LOG(DEBUG) << "Throttling BulkApply for " << absl::FromChrono(d);
```

Comment:
I think ideally we would have a callback here so we can capture time sleeping to report at a higher level then actual library (or expose an accessor for this information).

Reply:
Ok. We left the option to enable it as `experimental`, so we can change it to better support your use case.

So logging is not enough observability? Does your application want to make real time decisions based on the throttling?

It would be easy enough to take some object from the application and hold it in the limiter. We will just need to decide what that object is. Let's continue the discussion internally.

```
namespace google::cloud::bigtable::experimental {
struct BulkApplyThrottlingConfig {
  // TODO : what goes in here? e.g. a callback?
  std::function<void(std::chrono::system_clock::duration)> callback_ = [] (auto d) {};
};

struct BulkApplyThrottlingOption {
  using Type = BulkApplyThrottlingConfig;
};
}
```